packages/button/package.json

```diff
@@ -16,5 +19,14 @@
   "repository": {
     "type": "git",
     "url": "git+https://github.com/maecapozzi/constellation.git"
+  },
+  "dependencies": {
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "reakit": "^1.1.2"
```

Comment:
Wasn't sure if reakit should be a peer dependency or not. Thoughts?

Reply:
Hmm, I don't think they should be regular dependencies but now I'm unsure about whether they should be peer.

Reply:
I'm pretty sure react and react-dom should be peer deps.

Reply:
actually, we probs don't even need react-dom